specification/agrifood/data-plane/Microsoft.AgFoodPlatform/preview/2023-04-01-preview/agfood.json

```diff
@@ -1087,6 +1087,12 @@
             "description": "Original File Name for this attachment.",
             "type": "string"
           },
+          {
+            "in": "formData",
+            "name": "partyId",
```

Comment:
why these form fields are being reflected in the diff? Intentional code changes ?